test/repos/bom.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex; rm -rf repo; mkdir repo; cd repo
+
+git init
+
+cat <<EOT >> windowslf.txt
+asdf
+asdf
+EOT
+
+unix2dos windowslf.txt
```

Comment:
I don't have this program locally so if we ever include this in a unit test we'll need to either find a native solution or make CI download the program. I won't let that hold up merging though